src/langserver/main.rs

```diff
@@ -6,40 +6,48 @@
 //! * https://microsoft.github.io/language-server-protocol/specification
 //! * https://github.com/rust-lang-nursery/rls
 #![deny(unsafe_code)]
-#![cfg_attr(not(target_arch="wasm32"), forbid(unsafe_code))]
+#![cfg_attr(not(target_arch = "wasm32"), forbid(unsafe_code))]
 
-extern crate url;
 extern crate serde;
 extern crate serde_json;
-#[macro_use] extern crate serde_derive;
-extern crate petgraph;
+extern crate url;
+#[macro_use]
+extern crate serde_derive;
+extern crate dreamchecker;
+extern crate dreamcompiler;
+extern crate dreammaker as dm;
 extern crate interval_tree;
-extern crate lsp_types as langserver;
 extern crate jsonrpc_core as jsonrpc;
-extern crate dreammaker as dm;
-extern crate dreamchecker;
+extern crate lsp_types as langserver;
+extern crate petgraph;
+extern crate windows_named_pipe;
 
-#[macro_use] mod macros;
-mod io;
+use std::io::prelude::*;
+use std::path::Path;
+use windows_named_pipe as Pipe;
```

Comment:
it's fine if the debugging functionality is restricted to Windows only but it shouldn't break the build on Linux